app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i .freeze
+  validates :name, presence: true
+  validates :email, presence: true, length: {maximum: 255},
+  format: {with: VALID_EMAIL_REGEX}
```

Comment:
1. Sai indent nhé 
```suggestion
  validates :email, presence: true, length: {maximum: 255},
    format: {with: VALID_EMAIL_REGEX}
```

2. Các giá trị như255 thì em khai báo vào file setting hì. để có file settings thì tìm hiểu gem config nhé 

Đặt tên biến thì nên dạng là `digit.digit_255` để đọc là biết giá trị của nó và đồng thời các thể tái sử dụng nhé